parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -154,13 +154,13 @@ pub mod pallet {
 
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config> {
-		pub initial_sync: InitialSyncOf<T>,
+		pub initial_sync: Option<InitialSyncOf<T>>,
```

Comment:
Not sure how `Option::Some` gets serialized using serde, but if necessary you may need to update the following line to get the E2E stack up:  https://github.com/Snowfork/snowbridge/blob/97a76facc7c43addb5cc94f146e3b187e30b841b/core/packages/test/scripts/helpers/mutateSpec.js#L22

Reply:
The E2E stack works & the tests pass (aside from the XCM ones), so it seems fine for now.